### DISCUSSION TITLE
The need for Kotlin code completion support

### DISCUSSION THREAD

**Discussion by binx6** [AndroidIDEOfficial/AndroidIDE#1595] [Ideas]
**This is very necessary, on the one hand, it can improve the efficiency of writing Kotlin code, and on the other hand, it can make AndroidIDE more user-friendly and practical. And forgive me for digressing here. The logging and visualizer of AndroidIDE are not done well enough, especially since logging has been criticized. Whether it is application logs or IDE logs. It is recommended to optimize more.**

**Comment by binx6** [comment]
@itsaky

**Comment by itsaky** [comment]
Kotlin language support is planned (#211), but it will take some time to be implemented.
